src/components/Slider/Slider.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+  import { onMount } from 'svelte';
+  import { spring } from 'svelte/motion';
+  import { asSlider } from '../../actions/slider';
+  import {
+    createSlidesXPoints,
+    getIncreasedSlideIndex,
+    getDecreasedSlideIndex
+  } from './helpers';
+  import { setWidthToElement } from '../../helpers/common';
+
+  const x = spring(0, {
+    stiffness: 0.07,
+    damping: 0.35
+  });
+
+  let slider;
+  let sliderContent;
+  let slidesCount;
+  let xOnSlideStart;
+  let slidesXPoints;
+  let activeSlideIndex = 0;
+  // to calculate x on slidemove
+  let currentTranslateX = 0;
+
+  onMount(() => {
+    window.addEventListener('resize', updateXPoints);
+
+    initializeSlider();
+  });
+
+  function updateXPoints() {
+    slidesXPoints =
+      createSlidesXPoints(slider, sliderContent, slidesCount) || slidesXPoints;
+
+    slidesXPoints.length && setX(slidesXPoints[activeSlideIndex]);
+  }
+
+  function initializeSlider() {
+    slidesCount = sliderContent.children.length;
+
+    setWidthToElement(`${80 * slidesCount}%`)(sliderContent);
+
+    updateXPoints();
```

Comment:
event base model is very missing here )
Well, actually we have an observable store in Scelte.
How do you think could we use it somehow?